config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
Nobody says you are not supposed to. I could imagine mounting this into the backoffice, so we can look at it without having to start it up locally. I don't see why we shouldn't keep this option.